src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -311,6 +331,7 @@ public String getDisplayName() {
         public ManualCondition newInstance(StaplerRequest req, JSONObject formData) throws FormException {
             ManualCondition instance = new ManualCondition();
             instance.users = formData.getString("users");
+            instance.executeButtonLabel = instance.getExecuteButtonLabel();
```

Comment:
This field is referred in following file
src/main/resources/hudson/plugins/promoted_builds/conditions/ManualCondition/index.jelly